share/wrf_timeseries.F

```diff
@@ -177,7 +181,6 @@ SUBROUTINE calc_ts_locations( grid )
          grid%ntsloc_domain = ntsloc_temp
    
          DO k=1,grid%ntsloc_domain
-   
```

Comment:
I prefer having whitespace between an executable line of code and a comment line, but that's just my opinion.

Reply:
@mgduda 
I don't see harm in having the space. I'll add it back in.